docs/UserGuide.md

```diff
@@ -144,6 +144,44 @@ Examples:
 * `list` followed by `delete 2` deletes the 2nd person in the address book.
 * `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
 
+### Scheduling a meeting : `schedule`
+
+Schedules a meeting with the specified person.
+
+Format: `schedule INDEX s/START_DATETIME e/END_DATETIME`
```

Comment:
Noticed that prefix is the same as `EMAIL` prefix. As of now there is no commands that `END_DATETIME` is shared with `EMAIL`. Is it fine to use the same prefix, or is it better to change the prefix now? @Dethada @Javiery3889 @YuZizhen @Arixeyeion

Reply:
Perhaps it will be better to use the same prefix, as each command will have its own set of prefixes